.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+	
+[*]
+indent_style = tab
+trim_trailing_whitespace = false
```

Comment:
Probably needs 
```
trim_trailing_whitespace = true
insert_final_newline = true
``` 
if you meant to PR this file.

Reply:
Yes I meant it to be in the PR. Added the changes.